content/preregistration.md

```diff
@@ -0,0 +1,27 @@
+# Preregistration
+
+Predictions are tested on data; preregistration helps prove that predictions precede the data. This chronology is essential for valid tests of predictions (confirmatory research). Preregistration is not necessary for research that has no predictions and uses no hypothesis testing (exploratory research). Preregistration derives its name from the act of registering predictions before (pre) analysis (not to be confused with [Preprints](preprints.md)).
+
+To show the necessity of preregistration, consider this analogy: Horse racing bets cannot be placed after the results are known. Predictions in research are bets on how well a phenomenon is understood; the more specific the prediction the higher the gains if correct. However, it also means that bets cannot change, even in the slightest sense. 
+
+By not preregistering predictions, there is no guarantee that what the prediction is said to be actually was the original prediction. This includes guarantees to the outside world but also to yourself (it is very easy to fool yourself). [A survey](https://doi.org/10.1177/0956797611430953) indicates that certain behaviors that could undermine researcher's own predictions seem to be the norm. 
+
+By preregistering predictions (hypotheses), it is possible to make research more rigorous. However, the benefits of preregistration depend on how thoroughly it is conducted. Moreover, a preregistration is only as worthwhile as the follow through during analysis and reporting of results. As such, doing a preregistration requires the researcher to be as specific as possible and to also include statements that remove all confusion. A good rule of thumb: Preregistrations need to be as specific as the analyses that will be conducted on top of them later.
+<!-- Might consider adding preregistration tips at a later stage -->
+
+There are several places on the Web to preregister hypotheses, including the [Open Science Framework](https://osf.io), [Zenodo](https://zenodo.org), or any other service that archives files reliably and with clear timestamps. Such services are currently always third-parties and need to be trusted to provide reliable information and preserve the information into the future (see also [Reference Rot](reference-rot.md) and [Preservation](preservation.md)). Currrently the [Preregistration Challenge](https://cos.io/prereg/) rewards individuals if they preregister a study with $1,000.
```

Comment:
Curr__r_ently the [Preregistration Challenge](https://cos.io/prereg/) rewards individuals if they preregister a study with $1,000.

I would nuance- Currently individuals preregistering a study _can_ be rewarded with $1,000 by the PreReg Challenge.

Reply:
Adopted!